tsl/test/sql/continuous_aggs_deprecated.sql

```diff
@@ -0,0 +1,1232 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+-- initialize the bgw mock state to prevent the materialization workers from running
+\c :TEST_DBNAME :ROLE_SUPERUSER
+
+CREATE OR REPLACE FUNCTION ts_bgw_params_create() RETURNS VOID
+AS :MODULE_PATHNAME LANGUAGE C VOLATILE;
+
+CREATE OR REPLACE FUNCTION test.continuous_aggs_find_view(cagg REGCLASS) RETURNS VOID
+AS :TSL_MODULE_PATHNAME, 'ts_test_continuous_agg_find_by_view_name' LANGUAGE C;
+
+\set WAIT_ON_JOB 0
+\set IMMEDIATELY_SET_UNTIL 1
+\set WAIT_FOR_OTHER_TO_ADVANCE 2
+
+-- remove any default jobs, e.g., telemetry so bgw_job isn't polluted
+DELETE FROM _timescaledb_config.bgw_job WHERE TRUE;
```

Comment:
Hmmm... didn't pay attention on that because I just made a copy of the current `continuous_agg.sql` regression tests... will remove because is useless